### PR TITLE
[exp] optionally control network

### PR DESCRIPTION
**Pull request by czkkkkkk**

### CODE REVIEW

File: include/husky-base/threadsafe_priority_queue.h [hunk 1]
```diff
@@ -15,11 +18,12 @@ namespace base {
 // Handle smaller priority event first
 // Events with same priority should be FIFO
 // By now, priority = -1 are system mailbox events, other priority >= 0 are job events
-// TODO(czk) send BinStream of different jobs evenly
 template <typename T>
 class ThreadsafePriorityQueue {
  public:
-  ThreadsafePriorityQueue() = default;
+  ThreadsafePriorityQueue(int sch_mode = 0) : sch_mode_(sch_mode) {
+    std::cerr << "Sch mode " << sch_mode_ << std::endl;
```
Comment (kygx-legend) [inline include/husky-base/threadsafe_priority_queue.h]: For debug?

Comment (czkkkkkk) [reply include/husky-base/threadsafe_priority_queue.h]: Fixed it

File: src/mailbox_sender.cc [hunk 2]
```diff
@@ -25,8 +26,10 @@ namespace base {
 MailboxSender::MailboxSender(const MailboxAddressBook& addr_book, zmq::context_t* zmq_context)
     : zmq_context_(zmq_context), addr_book_(addr_book) {
   addr_book_.ForEach([=](int process_id, const std::string& addr) {
+    std::cerr << "connect to " << addr << std::endl;
```
Comment (kygx-legend) [inline src/mailbox_sender.cc:29]: ditto

Comment (czkkkkkk) [reply src/mailbox_sender.cc:29]: This one is for debug purpose. Sometime worker may fail in this scope.

**Comment by kygx-legend** [comment]
`CI` build failed. `error: ‘cerr’ is not a member of ‘std’`. Please add include file.

**Comment by czkkkkkk** [comment]
Fixed..

**Pull request merged**